### PR TITLE
Update spellcheck and expand what we check

### PR DESCRIPTION
**Pull request by tas50**
Check the yaml too

### CODE REVIEW

File: config/crd/bases/k8s.mondoo.com_mondooauditconfigs.yaml [hunk 1]
```diff
@@ -103,7 +103,7 @@ spec:
                       include:
                         description: Include is the list of resources to watch/scan.
                           Setting Include overrides anything in the Exclude list as
-                          specifying an Include list is effectively excluding everying
+                          specifying an Include list is effectively excluding everything
```
Comment (imilchev) [inline config/crd/bases/k8s.mondoo.com_mondooauditconfigs.yaml:106]: this is auto-generated file. I will fix this in the code

Comment (imilchev) [reply config/crd/bases/k8s.mondoo.com_mondooauditconfigs.yaml:106]: ah you did that already! nice!

**Review by imilchev — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -17,10 +17,13 @@ The Mondoo Operator provides the following features:
 - Continuous validation of Kubernetes nodes **without** privileged access
 - Admission Controller
 
-It is backed by Mondoo's powerful Policy-as-Code the Mondoo Query Language (MQL). Mondoo ships out-of-the-box security policies for Kubernetes:
+It is backed by Mondoo's powerful Policy-as-Code engine [cnspec](https://mondoo.com/docs/cnspec/cnspec-about/) and [MQL](https://mondoo.com/docs/mql/resources/). Mondoo ships out-of-the-box security policies for:
```
Comment (misterpantz) [inline README.md:20]:
```suggestion
It is backed by Mondoo's powerful policy-as-code engine [cnspec](https://mondoo.com/docs/cnspec/cnspec-about/) and [MQL](https://mondoo.com/docs/mql/resources/). Mondoo ships out-of-the-box security policies for:
```

File: config/manifests/patches/description.yaml [hunk 3]
```diff
@@ -6,21 +6,24 @@
     The Mondoo Operator provides the following features:
 
     - Continuous validation of deployed workloads
-    - Continuous validation of Kubernetes nodes **without** priviledged access
+    - Continuous validation of Kubernetes nodes **without** privileged access
     - Admission Controller 
 
-    It is backed by Mondoo's powerful [Policy-as-Code](https://mondoo.com/docs/getstarted/policy-as-code) engine and [MQL](https://mondoo.com/docs/mql/resources/). Mondoo ships out-of-the-box security policies for:
+    It is backed by Mondoo's powerful Policy-as-Code engine [cnspec](https://mondoo.com/docs/cnspec/cnspec-about/) and [MQL](https://mondoo.com/docs/mql/resources/). Mondoo ships out-of-the-box security policies for:
```
Comment (misterpantz) [inline config/manifests/patches/description.yaml:12]:
```suggestion
    It is backed by Mondoo's powerful policy-as-code engine [cnspec](https://mondoo.com/docs/cnspec/cnspec-about/) and [MQL](https://mondoo.com/docs/mql/resources/). Mondoo ships out-of-the-box security policies for:
```

**Review by misterpantz — APPROVED** [review]
yay!

**Review by czunker — APPROVED** [review]

**Pull request merged**